tasks/bottle.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const shell = require('shellpromise');
+const semver = require('semver');
+
+function npmBottle (increment) {//, isBeta, newModule) {
+	return shell('npm whoami')
+		.then(user => {
+			if (user.trim() !== 'financial-times') {
+				throw `Sign in to npm as financial-times before releasing anything.
+							Ask somebody about getting access to the account`;
+			}
+		})
+		.then(() => shell(`npm version ${increment}`))
+		.then(() => shell('npm publish'))
+		.then(() => shell('git push --tags origin HEAD'))
```

Comment:
nbt logs any uncaught errors once a task exits, so anything that shodul be logged to the user is just thrown